packages/material-ui/src/ButtonGroup/ButtonGroup.test.js

```diff
@@ -0,0 +1,229 @@
+import React from 'react';
+import { assert } from 'chai';
+import {
+  createMount,
+  createShallow,
+  createRender,
+  // describeConformance,
+  getClasses,
+} from '@material-ui/core/test-utils';
+import Button from '../Button/Button';
+import ButtonGroup from './ButtonGroup';
+
+describe('<ButtonGroup />', () => {
+  let mount;
+  let shallow;
+  let render;
+  let classes;
+
+  before(() => {
+    mount = createMount({ strict: true });
+    shallow = createShallow({ dive: true });
```

Comment:
Please remove all the shallow tests.

Reply:
That would mean we have no tests?

Reply:
Do you have a case that the mount can't cover?